draft-ietf-mls-protocol.md

```diff
@@ -4836,6 +4846,53 @@ be securely removed from a group. It also allows a member to rotate their
 keypair such that the old private key can no longer be used to decrypt new
 messages.
 
+## Confidentiality of Group Metadata
+
+MLS handles certain metadata about a group that is sometimes exposed to parties
+outside the group:
+
+* Group ID
+* Group membership
+* Group extensions
+* Frequency of epoch changes
+* Frequency of application messages within an epoch
```

Comment:
If we're using public handshake messages, it would also expose who is adding whom and when.